tools/depth-quality/depth-quality-model.cpp

```diff
@@ -46,8 +46,19 @@ namespace rs2
             {
                 if (valid_config = cfg.can_resolve(_pipe))
                 {
-                    _pipe.start(cfg);
-                    break;
+                    static bool device_is_used = false;
```

Comment:
The warning is implemented to be triggered only once in the application lifecycle.
Please refactor this from a static to a class member and add a reset flag functionality so it would be able to re-trigger this warning without restarting the tool